Source/DafnyCore/Compilers/Rust/Dafny-compiler-rust.dfy

```diff
@@ -2100,12 +2703,14 @@ module {:extern "DCOMP"} DCOMP {
 
     static method Compile(p: seq<Module>) returns (s: string) {
       s := "#![allow(warnings, unconditional_panic)]\n";
+      s := s + "#![allow(nonstandard_style)]\n";
```

Comment:
Kind of sums up the overall philosophy of our code generators nicely doesn't it. :)

Reply:
Note that smithy-rust does change CamelCase to snake_case, we could have a compiler option for doing that as well :-)